### PR TITLE
Adding landing page

### PR DESCRIPTION
**Pull request by b0llu**
## What to Check
- [ ] src/Pages/LandingPage/LandingPage.jsx

## What not to check
Do not Check Any Other Files

Live Preview,
https://stormkeep.netlify.app/

### CODE REVIEW

File: src/Pages/LandingPage/LandingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        </span>
+        <Link to="/signup">
+          <button className="landing-btn">JOIN NOW</button>
+        </Link>
+        <Link className="landing-link" to="/login">
+          <span>Already Have an account?</span>
+        </Link>
+      </div>
+      <div className="landing-right">
+        <img className="rsp-img" src="/assets/landing-img.svg" alt="" />
```
Comment (BalajiHarinath) [inline src/Pages/LandingPage/LandingPage.jsx:21]: Suggestion : give a value for alt attribute

Comment (b0llu) [reply src/Pages/LandingPage/LandingPage.jsx:21]: Thanks for pointing it out!

Comment (govindappaarun) [inline src/Pages/LandingPage/LandingPage.jsx:21]: add loading="lazy"

File: src/Pages/LandingPage/LandingPage.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  cursor: pointer;
+  font-size: 1.5rem;
+}
+
+.landing-right {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (govindappaarun) [inline src/Pages/LandingPage/LandingPage.css:58]: you could combine declarations for left and right, as they are almost same, save to reduce number of lines

**Review by govindappaarun — APPROVED** [review]

**Review by BalajiHarinath — APPROVED** [review]

File: src/Pages/LandingPage/LandingPage.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/LandingPage/LandingPage.jsx:21]: thanks for the suggestion

File: src/Pages/LandingPage/LandingPage.css (hunk 2 above)
Comment (b0llu) [reply src/Pages/LandingPage/LandingPage.css:58]: didint think about that I will try to implement it

**Pull request merged**